scripts/src/public/services/users.ts

```diff
@@ -15,18 +15,22 @@ import { assertRateLimitRegistration } from "../../utils/rate_limit";
 
 export type AuthToken = {
 	token: string;
-	activated: boolean;
-	expiration_date: string;
 	app_id: string;
 	user_id: string;
+	activated: boolean;
+	expiration_date: string;
 	ecosystem_user_id: string;
+	type: "new" | "returning";
```

Comment:
maybe `type` isn't a good name, but i don't want it to return as boolean, i like to keep the two options `"new"` and `"returning"`

Reply:
@liorama because this isn't just code, it's communication and data that is passed around, logged, etc. also, i can think of maybe other later values. but this is no longer relevant as i'll replace this with user info property instead.